docs/basic-usage.rst

```diff
@@ -491,6 +491,29 @@ alter the ``contain()`` list for the pagination query:
         return $this->Crud->execute();
     }
 
+Bulk Actions
+------------
+
+There are bulk actions which can be easily used with crud view.
+
+To set up crud action in controller do something like this in initialize method.
+
+.. code-block:: php
+    $this->loadComponent('Crud.Crud', [
+        'actions' => [
+            'deleteAll' => [
+                'className' => 'Crud.Bulk/Delete',
+            ],
+        ]
+    ]);
+
+Then you need to config scaffold to display select with actions.
+
+.. code-block:: php
+    $action->config('scaffold.bulk_actions', [
+        Router::url(['action' => 'deleteAll']) => __('Delete records'),
```

Comment:
I agree it would be cleaner, but it does not work that way. This bulk_actions config is passed into Form input as options parameter

See https://github.com/FriendsOfCake/crud-view/blob/master/src/Template/Element/index/bulk_actions/form_end.ctp